### PR TITLE
Added a new title to the existing list 

### PR DESCRIPTION
**Pull request by packtpartner**

### CODE REVIEW

**Comment by packtpartner** [comment]
Hi @liorze  I have added a new book in the documentation section, can you help me with it?

**Comment by liorze** [comment]
Sure, what can I do for you? 
Just letting you know, I'm not a committer in this project.

**Comment by packtpartner** [comment]
can u push the request? @liorze

**Comment by packtpartner** [comment]
and merge the request to the documents page

**Comment by packtpartner** [comment]
@jpeach  can you help me in this?

File: docs/home.md [hunk 1]
```diff
@@ -109,6 +109,11 @@ layout: documentation
 
 <div class="row">
   <div class="col-xs-6 col-md-4">
+    <a href="https://www.packtpub.com/big-data-and-business-intelligence/mastering-mesos" class="thumbnail">
```
Comment (bmahler) [inline docs/home.md]: I think the current ordering is alphabetical based on the name of the title, can you place this after "Building Applications..." and before "Mesos in Action..."?

File: docs/home.md [hunk 2]
```diff
@@ -109,6 +109,11 @@ layout: documentation
...
+      <img src="https://www.packtpub.com/sites/default/files/6249OS_5186%20Mastering%20Mesos.jpg" alt="Mastering Mesos by Dipa Dubhashi and Akhil Das">
+    </a>
+<div class="row">
```
Comment (bmahler) [inline docs/home.md]: Also it looks like you don't want to open a new class=row div here. You're missing the title text as well. If you have a look at the other entries you can see the expected format.

**Comment by bmahler** [comment]
I pushed this with some minor changes to make it consistent with the other entries.

**Comment by packtpartner** [comment]
Thank you @bmahler 
Sorry for the trouble.

**Comment by packtpartner** [comment]
@bmahler 
Just checking in

**Comment by kaysoky** [comment]
The book in this PR is already staged (i.e. committed).  We plan to update the website later.

**Comment by packtpartner** [comment]
@kaysoky thanks :)

**Pull request closed without merging**